### DISCUSSION TITLE
Is it possible to use Tables without a Model?

### DISCUSSION THREAD

**Discussion by michabbb** [Laravel-Backpack/community-forum#595] [Q&A (Help)] [answered]
lets say the data comes from a json api result....

thanks

**Comment by christmex** [comment]
Hi @michabbb I think you need to create your own custom list view, there are 2 ways
1. overwrite all of the list blade
2. choose your custom list inside your selected CrudController

The first way, you can achieve with this
```bash
// open your terminal and slap this code 
php artisan backpack:publish crud/list
```
after that you can start to do what you want, you can start to manipulate that file to achieve what you need
here's the reference: https://backpackforlaravel.com/docs/6.x/crud-operation-list-entries#how-to-overwrite-1

The second way you can achieve just with this line
```php
$this->crud->setListView('path.to.your.view');
```
Put that code inside your setupListOperation, after that just copy the backpack list.blade in ``` vendor/backpack/crud/src/resources/views/crud/list.blade.php ``` and start to modify it
here's the reference: https://backpackforlaravel.com/docs/6.x/crud-api#custom-views

I would say to choose the second way because I think you need just for a few pages, I hope you get this idea.


If you want i can try to create the custom view and back with the updates here, do you have example data, so I can use it to try to make the custom view😉


Cheers.

**Reply by michabbb** [reply]
Thanks a lot for that detailed answer 👍😎

**Comment by ziming** [comment, marked as answer]
you can try laravel sushi to make a "fake model"

it is from the creator of livewire

**Reply by pxpm** [reply]
Indeed. You can use Sushi.  We use it in DevTools and it's great, highly recommend. 

As @christmex said would also work, but I think it will be waaaaay more work than using Sushi.

Thanks @ziming 🙏 

If you need any help please let us know, but it should be very straightforward to use, I am not aware of something special you need to keep in mind.

**Reply by tabacitu** [reply]
![winning-homer-simpson-gif](https://github.com/Laravel-Backpack/community-forum/assets/1032474/1af020b3-357c-4595-bee5-37d0a32500f0)

I would also recommend using Sushi - https://github.com/calebporzio/sushi
